plugins/catalog-backend-module-aws/src/providers/config.ts

```diff
@@ -46,6 +46,7 @@ function readAwsS3Config(id: string, config: Config): AwsS3Config {
   const bucketName = config.getString('bucketName');
   const region = config.getOptionalString('region');
   const prefix = config.getOptionalString('prefix');
+  const accountId = config.getOptionalString('accountId');
```

Comment:
When you add new config parameters, they need to be added [to the config schema too](https://github.com/backstage/backstage/blob/de39377ee7994de780d9cfc030b714967c3525ef/plugins/catalog-backend-module-aws/config.d.ts#L40). Note that it should be added in two places in that file, preferably with some comment that explains its usage and significance.

Reply:
Sorry, forgot to update it. Now it is done!